--- sphinxrenderer.py	2017-06-16 15:17:38.177631776 +0200
+++ sphinxrenderer.py	2017-06-16 15:17:29.601594435 +0200
@@ -1091,7 +1091,8 @@
         #
         # If there is no description then render then term by itself
         if nodelist:
-            nodelist[0].children = [term, separator] + nodelist[0].children
+            nodelist[0].insert(0, term)
+            nodelist[0].insert(1, separator)
         else:
             nodelist = [term]
